packages/react/src/components/List/List.tsx

```diff
@@ -47,7 +46,6 @@ export interface IListState<T = any> {
   getDerivedStateFromProps(nextProps: IListProps<T>, previousState: IListState<T>): IListState<T>;
 
   pagesVersion?: {};
-  hasMounted: boolean;
```

Comment:
It looks like this interface is exported. Is it a breaking change to remove this?

Reply:
Yes it is! Re-adding it.